integration/kubernetes/k8s-volume.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	pod_config_dir="${BATS_TEST_DIRNAME}/untrusted_workloads"
+	path="/mnt/data"
+	msg="Hello from Kubernetes"
+	sudo -E mkdir $path
+	echo $msg > $path/index.html
+	pod_name="pv-pod"
+}
+
+@test "Create Persistent Volume" {
+	volume_name="pv-volume"
+	volume_claim="pv-claim"
+
+	# Create the persistent volume
+	sudo -E kubectl create -f "${pod_config_dir}/pod-pv-volume.yaml"
+
+	# Check the persistent volume
+	sudo -E kubectl get pv $volume_name | grep Available
+
+	# Create the persistent volume claim
+	sudo -E kubectl create -f "${pod_config_dir}/pod-volume-claim.yaml"
+
+	# Check the persistent volume claim
+	sudo -E kubectl get pvc $volume_claim | grep Bound
+
+	# Create pod
+	sudo -E kubectl create -f "${pod_config_dir}/pod-pv-pod.yaml"
+
+	# Check pod creation
+	sudo -E kubectl wait --for=condition=Ready pod "$pod_name"
+
+	cmd="apt-get -y update && apt-get install -y curl && curl localhost"
+	sudo -E kubectl exec $pod_name -- sh -c "$cmd" | grep "$msg"
+}
+
+teardown() {
+	sudo -E kubectl delete pod "$pod_name"
+	sudo rm -rf $path
```

Comment:
I'd say we should do the delete in a TRAP, so we cleanup on fail - but, maybe bats does that for the teardown func anyway?, and/or maybe bat doesn't support TRAP.

Reply:
`teardown` will be executed even if the test fails, so we are fine here.